tasks/copy_to_glacier/schemas/config.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_files/schemas/config.json",
```

Comment:
Shouldn't this be `copy_to_glacier` in the **id**, **title**, and **description**?

Reply:
Fixed